src/components/PeopleTablet/PeopleTable.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { NoPeopleOnServer } from '../NoPeopleOnServer';
+
+interface Props {
+  peoples: Person[];
+}
+export enum Sex {
+  MALE = 'm',
+  FEMALE = 'f',
+}
+
+export const PeopleTable: React.FC<Props> = (props) => {
+  const { peoples } = props;
+
+  const { peopleId } = useParams();
+
+  return (
+
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      {peoples.length === 0 ? (
+        <NoPeopleOnServer />
+      ) : (
+        <tbody>
+          {
+            peoples.map(people => (
```

Comment:
people is already plural, for singular you could use person :)

Reply:
> people is already plural, for singular you could use person :)

Good point